ui/components/app/snaps/snap-home-page/useSnapHome.js

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import { handleSnapRequest } from '../../../../store/actions';
+
+export function useSnapHome({ snapId }) {
+  const [loading, setLoading] = useState(true);
+  const [data, setData] = useState(undefined);
+  const [error, setError] = useState(undefined);
+
+  useEffect(() => {
+    let cancelled = false;
+    async function fetchPage() {
+      try {
+        setError(undefined);
+        setLoading(true);
+
+        const newData = await handleSnapRequest({
+          snapId,
+          origin: '',
+          handler: 'onHomePage',
+          request: {
+            jsonrpc: '2.0',
+            method: ' ',
```

Comment:
Why is the method value just space ` `?

Reply:
Yeah, we don't have a good way to send a request to a handler that does not require a request currently